lib/internal/errors.js

```diff
@@ -132,22 +132,164 @@ class SystemError extends makeNodeError(Error) {
   }
 }
 
+function createErrDiff(actual, expected, operator) {
+  var other = '';
+  var res = '';
+  var lastPos = 0;
+  var end = '';
+  var skipped = false;
+  const actualLines = util
+    .inspect(actual, { compact: false, depth: 10 }).split('\n');
+  const expectedLines = util
+    .inspect(expected, { compact: false, depth: 10 }).split('\n');
+  const msg = `Input A expected to ${operator} input B:\n` +
+        '\u001b[32m+ expected\u001b[39m \u001b[31m- actual\u001b[39m';
```

Comment:
We should probably be testing that unix color codes are supported on the current terminal.

Reply:
I don’t think we can test that but it might make sense to:
- Check whether `process.stderr` is a TTY
- Make coloring overridable in some way

Reply:
That would indeed be nice... I will see what I can do but I am not sure if we can properly detect that.